GameServerLib/Logic/Packets/PacketHandlers/Handlers/HandleGameNumber.cs

```diff
@@ -1,18 +0,0 @@
-﻿using ENet;
-using LeagueSandbox.GameServer.Logic.Packets;
-using LeagueSandbox.GameServer.Logic.Players;
-
-namespace LeagueSandbox.GameServer.Core.Logic.PacketHandlers.Packets
-{
-    class HandleGameNumber : IPacketHandler
-    {
-        private Game _game = Program.ResolveDependency<Game>();
-        private PlayerManager _playerManager = Program.ResolveDependency<PlayerManager>();
-
-        public bool HandlePacket(Peer peer, byte[] data)
```

Comment:
Looks like it is no longer used, GameNumber was sent in HandleKeyCheck but thats disabled too. In the original handlers dictionary this was commened out.